### ISSUE TITLE
Nan values on training

### ISSUE THREAD

**Issue by carlosh93** [tensorlayer/hyperpose#330]
Hello! I am training the OpenPose model with the default parameters given at the current state of the repository. However, I am obtaining nan values on training. Below is an example of my training logs.

Train iteration 34000 / 1000000: Learning rate 9.999999747378752e-05 total_loss:nan, conf_loss:nan, paf_loss:nan, l2_loss nan stage_num:6 time:0.0003819465637207031
stage_0 conf_loss:nan paf_loss:nan
stage_1 conf_loss:nan paf_loss:nan
stage_2 conf_loss:nan paf_loss:nan
stage_3 conf_loss:1271.6895751953125 paf_loss:nan
stage_4 conf_loss:nan paf_loss:nan
stage_5 conf_loss:nan paf_loss:nan

Do you have any idea why this is happening?

**Comment by Gyx-One** [comment]
Hello @carlosh93 !
Thanks for using our library.
This is one of the major issue that I encountered before when I try to train **Openpose** model, I think the large parameter number of **Openpose** model is the major cause. I tackled this by starting training from the converted official vgg pretrained backbone and it turns to convergent well :).
Last month I released the trained models and pretrained backbones, you may start with loading the **newest_vgg19_backbone.npz**. 
@ganler uploaded these models,  I think he knows where the trained model to be found. :)

**Comment by carlosh93** [comment]
Hello! Thank you for your response. I searched for the file you mentioned, but it seems that it does not exist within the repository nor the [google drive shared folder](https://drive.google.com/drive/folders/1w9EjMkrjxOmMw3Rf6fXXkiv_ge7M99jR). Would you please share the model? I need to train the openpose model. Or, if you have a pretrained openpose model (don't matter if it is not the final version), would you please share it? Thank you!

**Comment by ganler** [comment]
@carlosh93 Please check your email. I sent a copy to you. I am sorry that I haven't uploaded them to google drive yet as we met some problems during the PPN model conversion.(we want to submit all models(openpose(OK now) + ppn(WIP)) once)

**Comment by davogrego** [comment]
@ganler can you please send a copy to my email too?
